modules/cnv_install_1_4.adoc

```diff
@@ -0,0 +1,64 @@
+[[install_cnv]]
+=== Installing {ProductName} to {product-title}
+
+The `kubevirt-ansible` RPM contains the latest automation for deploying 
+{ProductName}. This procedure installs all {ProductName} components to your 
+{product-title} cluster. 
+
+This procedure installs the following components:
+
+* {ProductName} core components (KubeVirt)
+* Containerized data importer (CDI) controller
+* Multus, Open vSwitch (OVS), and SR-IOV container network interface plug-ins
+* Updated {product-title} web console
+
+.Prerequisites
+* A running {product-title} 3.11 cluster
+* User with `cluster-admin` privileges
+
+.Procedure
+
+. Log into the {product-title} cluster as an admin user:
++
+----
+$ oc login -u system:admin
+----
+
+. Install the `kubevirt-ansible` RPM and its dependencies:
++
+----
+$ yum install kubevirt-ansible
+----
+
+. If your cluster has unique requirements, customize the 
+`/usr/share/ansible/kubevirt-ansible/vars/cnv.yml` file. Otherwise, skip this 
+step.
+.. Copy the file to your local path:
++
+----
+$ cp /usr/share/ansible/kubevirt-ansible/vars/cnv.yml local-cnv.yml
+----
+.. Edit the copy:
++
+----
+$ vim local-cnv.yml
+----
+
+. Launch {ProductName}:
+** If you use the default `cnv.yml` file, run:
++
+----
+$ ansible-playbook -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml -e 
```

Comment:
At this point, we should expect that either the user is in `/usr/share/ansible/kubevirt-ansible` or in a local copy of this dir.

Reply:
For remote installs, we're going to use the inventory file that kubevirt-ansible ships with.  The user can edit it as needed.  So add to every `ansible-playbook` command a `-i inventory`.

Reply:
Adding "-i inventory" to both install and uninstall ansible-playbook commands.  

> At this point, we should expect that either the user is in `/usr/share/ansible/kubevirt-ansible` or in a local copy of this dir.

Is this still the case since we're having users specify any modifications on the command line?